cmd/kubeconfig/create.go

```diff
@@ -56,8 +56,12 @@ func NewCreateCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "A hostedcluster namespace")
 	cmd.Flags().StringVar(&opts.Name, "name", opts.Name, "A hostedcluster name")
+	allNamespaces := cmd.Flags().BoolP("all-namespaces", "A", false, "If kubeconfigs for clusters in all namespaces should be created")
```

Comment:
Seems like we could do this without adding a new flag by saying that the command does all namespaces by default and if you want to limit it to a namespace, you can with the existing `--namespace` flag.  If the code expects `--namespace` and `--name` to always be set together, we'd have to rework that.